client.go

```diff
@@ -54,18 +54,23 @@ func (c *Client) Close() error {
 
 // Request executes a STUN request against the clients server
 func (c *Client) Request() (*Message, error) {
+	return Request(c.conn)
+}
+
+// Request executes a STUN request using the supplied connection
+func Request(conn net.Conn) (*Message, error) {
```

Comment:
Linter issue: `` `conn` can be `io.ReadWriter` (interfacer) ``

Should I fix this? I originally used io.ReadWriter for this, but it makes more sense to me to keep it as net.Conn.

Reply:
Also, interfacer is deprecated because it is prone to bad suggestions: https://github.com/golangci/interfacer

Reply:
aye didn't know that about `Interface` is deprecated, I would just disable it then.

I will disable it every other repo tonight. We also need to add someway to auto-update golangci